lab/experiments/ActionBarTransition/ItemModal.vue

```diff
@@ -180,6 +181,7 @@ export default {
 			var(--regular-bottom-padding)
 			+ var(--extra-bottom-padding-for-deadclick)
 			+ var(--safe-area-inset-padding)
+			+ var(--chrome-bottom-offset, 0px)
```

Comment:
can you add a comment here, and everywhere else where `--chrome-bottom-offset` is used to explain that this is a temp fix / undefined var that comes from outside of Maker? thanks